apps/factory/src/components/create_template.tsx

```diff
@@ -30,24 +30,28 @@ import { FormDropDownInput } from "./form_components/formDropDownInput";
 import EditIcon from "@mui/icons-material/Edit";
 import { api } from "~/utils/api";
 import toast from "react-hot-toast";
-// import * as Sentry from "@sentry/nextjs";
+import { useRouter } from "next/router";
 export function CreateTemplate({
   pp,
   onCreate,
   sx,
   status,
   customError,
   ptId,
+  cube,
+  edit,
 }: {
   pp: pp;
   onCreate: Function;
   sx?: any;
   status: string;
   customError: any;
   ptId: string | boolean;
+  cube?: boolean;
+  edit?: string | string[];
```

Comment:
@Rahul917-Kumar please explain use of edit field, not sure why it need to be passed ?